rasa/shared/utils/common.py

```diff
@@ -205,3 +206,13 @@ def arguments_of(func: Callable) -> List[Text]:
     import inspect
 
     return list(inspect.signature(func).parameters.keys())
+
+
+def raise_on_unexpected_train(f):
```

Comment:
Let's please add a docstring which points them to the docs - also worth updating the #product-engineering channel about it